src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -0,0 +1,173 @@
+import { mount, shallow } from "enzyme"
+import React from "react"
+
+import { OrderWithShippingDetails } from "Apps/__test__/Fixtures/Order"
+import { Checkbox } from "../../../../Styleguide/Elements"
+import { AddressForm } from "../../Components/AddressForm"
+import { ContinueButton, PaymentProps, PaymentRoute } from "../Payment"
+
+jest.mock("react-relay", () => ({
+  commitMutation: jest.fn(),
+  createFragmentContainer: component => component,
+}))
+
+import { commitMutation, RelayProp } from "react-relay"
+import { creatingCreditCardSuccess } from "../../../__test__/Fixtures/MutationResults/createCreditCard"
+import { settingOrderPaymentSuccess } from "../../../__test__/Fixtures/MutationResults/setOrderPayment"
+
+const mutationMock = commitMutation as jest.Mock<any>
+
+describe("Payment", () => {
+  let stripeMock
+
+  const mountPaymentRoute = props => {
+    const providerComponent = shallow(<PaymentRoute {...props} />)
+    const responsiveComponent = shallow(
+      providerComponent
+        .childAt(2)
+        .props()
+        .children({ xs: false })
+    ) as any
+
+    return shallow(responsiveComponent.props().children({ xs: false }))
```

Comment:
This is the questionable function that shallow-mounts the component. We don't want to mount the component entirely otherwise Stripe components will fail due to the lack of `context` values that are supposed to be injected. The library's test does seem to [test them with context values injected properly](https://github.com/stripe/react-stripe-elements/blob/732dd3ae7c64f171218064af012fbc2b13842482/src/components/Element.test.js#L38), but I couldn't figure out how we could do the same in here.